bin/commandTopicConsumer.py

```diff
@@ -0,0 +1,165 @@
+    global max_offset
+    for p in partitions:
+        high_water = consumer.get_watermark_offsets(p)[1]
+        if high_water >= max_offset:
+            max_offset = high_water
+
+class CommandConsumer(object):
+    def __init__(self, ksqlServiceId, conf):
+        self.consumer = Consumer(conf)
+        self.topic = '_confluent-ksql-{}_command_topic'.format(ksqlServiceId)
+
+    def consumer_run(self):
+        global max_offset
+        self.consumer.subscribe([self.topic], on_assign=consumer_latest_offsets)
+        #print("max offset = ", max_offset)
+        self.msg_cnt = 0
+        self.msg_err_cnt = 0
+        stmts = {}
+        try:
+            while True:
+                msg = self.consumer.poll(0.2)
+                if msg is None:
+                    continue
+
+                if msg.error() is not None:
+                    print("consumer: error: {}".format(msg.error()))
+                    self.consumer_err_cnt += 1
+                    continue
+
+                try:
+                    #print("Read msg with offset ", msg.offset())
+                    self.msg_cnt += 1
+                    record = CommandRecord.deserialize(msg.value())
+                    match = re.search(r'(?:create|drop) (?:stream|table) ([a-zA-z0-9-]+?)(:?\(|AS|\s|;)', record.stmt, re.I)
+                    if match:
+                        name = match.group(1).upper()
+                        if name == "KSQL_PROCESSING_LOG":
+                            continue
+                        if name not in stmts:
+                            stmts[name] = []
+                        stmts[name].append(record.stmt)
+                    else:
+                        match2 = re.search(r'(?:terminate) (?:ctas|csas)_(.+?)_', record.stmt, re.I)
+                        if match2:
+                            name = match2.group(1).upper()
+                            stmts[name].append(record.stmt)
+                        else:
+                            print("No match %s",record)
+                    # High watermark is +2 from last offset
+                    if msg.offset() >= max_offset-2:
+                        break
+
+                except ValueError as ex:
+                    print("consumer: Failed to deserialize message in "
+                                     "{} [{}] at offset {} (headers {}): {}".format(
+                                         msg.topic(), msg.partition(), msg.offset(), msg.headers(), ex))
+                    self.msg_err_cnt += 1
+
+        except KeyboardInterrupt:
+            pass
+
+        finally:
+            self.consumer.close()
+            print("Consumed {} messages, erroneous message = {}.".format(self.msg_cnt, self.msg_err_cnt))
+            for key, value in stmts.items():
+                print("[{}]".format(key))
+                for cmd in value:
+                    print("> {}".format(cmd))
+                print("-------------------------------------------------\n")
```

Comment:
I wonder if instead we can actually print this in a machine parseable format (e.g. json):
```json
[
  {
    "subject": "PAGEVIEWS",
    "statements": [
      "CREATE TABLE FOO WITH (KAFKA_TOPIC='FOO', ...",
      "DROP ...
    ]
  },
  ...
]
```

Reply:
and/or in some way that makes it easy to just paste into the CLI/UI. It's probably worth adding an output option.